README.md

```diff
@@ -142,13 +169,13 @@ function cholesky_dft!(A, ts)
         # Left blocks update
         U = UpperTriangular(T[i,i])
         for j in i+1:n
-            @dspawn ldiv!(@R(U)', @RW(T[i,j])) label="ldiv ($i,$j)"
+            @dspawn ldiv!(@R(U'), @RW(T[i,j])) label="ldiv ($i,$j)"
```

Comment:
I thought putting the data dependency on `U'` rather than `U` came with a performance issue... Is that not the case any more?